utils/GenerateSpritesheet.java

```diff
@@ -16,17 +16,27 @@
  * The images are called "PREFIX0000.png" through "PREFIX9999.png", where `PREFIX` may be different for each of the four rotations
  * (let's call the prefixes `se`,`ne`,`nw`,`sw` in this order). The number of the
  * animation's starting frame is `idOff_<p>` for each of the four prefixes <p>.
+ *
  * To generate the spritesheet and save it as `output_file`, run
- *    java GenerateSpritesheet src_dir output_file idOff_se se idOff_ne ne idOff_nw nw idOff_sw sw f x y w h true
- * The last, optional parameter can be set to `false` to disable intelligent calculation of frame indices
- * (useful when (ab)using this tool to generate spritesheets for a different purpose than the one stated above).
+ *    java GenerateSpritesheet src_dir output_file idOff_se se idOff_ne ne idOff_nw nw idOff_sw sw f x y w h TIMA
+ *
+ * The last parameter declares which indexing algorithm to use. The indexing algorithm decides at which coordinates
+ * in the resulting spritesheet each input image should be located. Available indexing algorithms are:
+ * - TIMA: Arranges the individual images in the way expected by the TIMA and FSET node generators.
+ * - PLAIN: Just put one sprite after the other, without considering rotations or other fancy processing.
```

Comment:
If you stick that in a Java doc with the constants, an IDE will display that whenever you come near the value, giving your efforts much more impact.